src/app/composer/qgscomposermanager.cpp

```diff
@@ -338,101 +389,86 @@ void QgsComposerManager::changeEvent( QEvent* event )
 
 void QgsComposerManager::remove_clicked()
 {
-  QListWidgetItem* item = mComposerListWidget->currentItem();
-  if ( !item )
-  {
-    return;
-  }
+  QList<QgsComposer *> composersList;
+  QList<QListWidgetItem *> composersItems = mComposerListWidget->selectedItems();
+  QString title = tr( "Remove composers" );
+  QString message = tr( "Do you really want to remove all these map composers ?" );
```

Comment:
A better string would be "Do you really want to remove all selected map composers?"